BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -270,6 +270,7 @@
   gPlatformModuleTokenSpaceGuid.PcdSmbiosTablesBase  | 0x00000000
   gPlatformModuleTokenSpaceGuid.PcdSmbiosStringsPtr  | 0x00000000
   gPlatformModuleTokenSpaceGuid.PcdSmbiosStringsCnt  | 0x0000
+  gPlatformModuleTokenSpaceGuid.PcdPlatformCpuInitHook | 0x00000000

```

Comment:
Can we rename it to PcdFuncCpuInitHook ?   For PCD that used as a function pointer,  we can let it follow  PcdFunc????? to indicate it is a function pointer.

Reply:
Good point. will do it